### DISCUSSION TITLE
Custom output width dynamic relative to portrait or landscape

### DISCUSSION THREAD

**Discussion by Craigjw** [upscayl/upscayl#1017] [Ideas]
Using the option to use a custom width, Portrait and landscape images are upscaled to different total resolutions
Can a dynamic option to have the image upscaled to the desired pixels dependent on portrait or landscape.
currently using width = 1600
800 x 600 = 1600 x 1200
600 x 800 = 1600 x 2100

suggested dynamic:
800 x 600 = 1600 x 1200
600 x 800 = 1200 x 1600

I hope that this makes sense as it'd be a really useful addition.

**Comment by eridevear** [comment]
![Uploading 34a0c9ed4debed885f49035ffb382367_0.jpeg…]()

**Comment by eridevear** [comment]
![430a953bed0794d7f30d394a9b32bbc0_0](https://github.com/user-attachments/assets/e2ed2d3b-3b42-4e8c-9484-3b02608f082a)
